src/components/MoneyRaised.js

```diff
@@ -31,9 +42,39 @@ const MoneyRaised = (props) => {
   }, msPerPenny)
 
   return (
-    <span className={classes.currencyText}>
-      {currencyFormatUSD(currentMoneyRaised)}
-    </span>
+    <div>
+      <Button ref={spanRef} onClick={() => setIsPopoverOpen(true)}>
```

Comment:
The button adds some additional margin that increased the menu spacing. Let's compensate for it so the menu stays approx the same: I think you can remove the `moneyRaisedContainer` class margin in index.js.